source/databases-collections.txt

```diff
@@ -0,0 +1,272 @@
+.. _scala-databases-collection:
```

Comment:
Small typo here that doesn't match the ref link in the index page:
```suggestion
.. _scala-databases-collections:
```

Reply:
Nice catch!